### PR TITLE
Roman add retries

### PR DESCRIPTION
**Pull request by romanbracinik**
BIG-112

### CODE REVIEW

File: src/BigQueryClientHandler.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            try {
+                return $this->client->send($request);
+            } catch (RequestException $e) {
+                if ($retries >= $this->maxRetries) {
+                    throw $e;
+                }
+            }
+            usleep(500000); // wait for 0.5 seconds before retrying
+            $retries++;
+        } while ($retries <= $this->maxRetries);
```
Comment (zajca) [inline src/BigQueryClientHandler.php]: Proč nepoužiješ https://github.com/keboola/retry

Comment (romanbracinik) [reply src/BigQueryClientHandler.php]: Jo to kludne mozem toto som C&P zo sapi aby som vyskusal ci to pomoze

**Comment by romanbracinik** [comment]
Tu ten build presiel este som nieco upravil https://github.com/keboola/php-storage-driver-bigquery/actions/runs/5813184363/job/15760177104?pr=53 ale davam to na review

**Review by zajca — APPROVED** [review]
Luks gut

**Pull request merged**